.github/workflows/nightly_build.yml

```diff
@@ -0,0 +1,43 @@
+name: Publish Nightly Release
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # This runs the workflow nightly at midnight UTC
+  workflow_dispatch: # Allows manual triggering of the workflow
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+       
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Run PowerShell script
+        shell: pwsh
+        run: ./nightly_build.ps1
+
+      - name: Create Release
```

Comment:
Creating a new release nightly would quickly spam the releases page - it would probably make more sense for it to just edit the file into the latest release rather than making a whole new release every time

Reply:
Yeah I was thinking about that, I will change it so it edits the same release.